src/dcos_e2e/cluster.py

```diff
@@ -185,10 +182,65 @@ def default_ssh_user(self) -> str:
         """
         return self._default_ssh_user
 
+    def install_dcos_from_url(
+        self,
+        build_artifact: str,
+        extra_config: Dict[str, Any] = None,
+        log_output_live: bool = False,
+    ) -> None:
+        """
+        Args:
+            build_artifact: The URL string to a build artifact to install DC/OS
+                from.
+            extra_config: Implementations may come with a "base"
+                configuration. This dictionary can contain extra installation
+                configuration variables.
+            log_output_live: If `True`, log output of the installation live.
+                If `True`, stderr is merged into stdout in the return value.
+
+        Raises:
+            NotImplementedError: `NotImplementedError` because the given
+                backend provides a more efficient installation method than
+                the DC/OS advanced installation method.
+        """
+        self._cluster.install_dcos_from_url(
+            build_artifact,
```

Comment:
Please use named arguments where possible. This is a personal preference of mine and not necessarily a best practice 😄